fe/fe-core/src/main/java/com/starrocks/common/Config.java

```diff
@@ -96,7 +96,7 @@ public class Config extends ConfigBase {
      * Log to file by default. set to `true` if you want to log to console
      */
     @ConfField
-    public static boolean sys_log_to_console = false;
+    public static boolean sys_log_to_console = System.getenv("SYS_LOG_TO_CONSOLE").trim().equals("1");
```

Comment:
looks like it doesn't work for this way to init the `sys_log_to_console`, need to take a look how to make it work or find an alternative way to do it.

Reply:
`System.getenv("SYS_LOG_TO_CONSOLE")` can be `null` if the env is not set.

Reply:
@kevincai fixed the issue. 
verified the following test cases
1)SYS_LOG_TO_CONSOLE env variable not set, sys_log_to_console=false
2)SYS_LOG_TO_CONSOLE env variable is present, start fe without --logconsole, sys_log_to_console=false
3)SYS_LOG_TO_CONSOLE env variable is present, start fe with --logconsole, sys_log_to_console=true
4)SYS_LOG_TO_CONSOLE not set from start_fe.sh, SYS_LOG_TO_CONSOLE=1 ./start_fe.sh, sys_log_to_console=true
5)SYS_LOG_TO_CONSOLE not set from start_fe.sh, SYS_LOG_TO_CONSOLE=0 ./start_fe.sh, sys_log_to_console=false
6)SYS_LOG_TO_CONSOLE not set from start_fe.sh, export SYS_LOG_TO_CONSOLE=1; ./start_fe.sh, sys_log_to_console=true
7)SYS_LOG_TO_CONSOLE not set from start_fe.sh, export SYS_LOG_TO_CONSOLE=0; ./start_fe.sh, sys_log_to_console=false

Reply:
looks like not make sense to me for case 2) 3) 4) 6).

From the code, it always `export SYS_LOG_TO_CONSOLE=${RUN_LOG_CONSOLE}`, if --logconsole is not provided, it will be always `export SYS_LOG_TO_CONSOLE=0` which overwrites the outside `SYS_LOG_TO_CONSOLE` env var before running the `start_fe.sh`

Maybe we should consider

```bash
RUN_LOG_CONSOLE=${SYS_LOG_TO_CONSOLE:-0}
```

in line 38?

Reply:
what i did was comment the setting in start_fe.sh and run the test
commented this line in start_fe.sh and run the test
#export SYS_LOG_TO_CONSOLE=${RUN_LOG_CONSOLE}

Reply:
but we have to put all together and do the testing, can't expect the user to manually comment out the line from `start_fe.sh`

Reply:
@kevincai i have fixed this.